torch/nn/modules/activation.py

```diff
@@ -895,6 +895,14 @@ def _arg_requires_grad(x: Optional[torch.Tensor]) -> bool:
     return False
 
 
+def _is_make_fx_tracing():
```

Comment:
Why is this code in torch.nn ?
That doesn't look like the right place to put it and there are 100% chance this is going to get broken if proxy mode is modified as no-one will expect to update this file.
Could you please move this utility within fx?

Reply:
FYI @bdhirsh

Reply:
Yeah, it is a hack suggested by @bdhirsh to temporarily unblock Executorch during model tracing. #107014 seems to plan to handle this in a nicer way.

Reply:
That other PR is not removing this utility though?

Reply:
Agree with @albanD we should mvoe this within fx. @albanD any suggestions on where it can go?

Reply:
I will defer to @bdhirsh for the final answer but next to the make_fx API would be my first suggestion